Library/Homebrew/exceptions.rb

```diff
@@ -555,12 +555,11 @@ def initialize(resource)
 # raised when a single patch file is not found and apply hasn't been specified
 class MissingApplyError < RuntimeError; end
 
-class BottleVersionMismatchError < RuntimeError
-  def initialize(bottle_file, bottle_version, formula, formula_version)
+class BottleFormulaUnavailableError < RuntimeError
+  def initialize(formula_path)
     super <<-EOS.undent
-      Bottle version mismatch
-      Bottle: #{bottle_file} (#{bottle_version})
-      Formula: #{formula.full_name} (#{formula_version})
+      This bottle does not contain the formula file:
```

Comment:
How about outputting the bottle file and expected formula path within it?

Reply:
Done.